samples/c/text_generation/README.md

```diff
@@ -42,6 +42,12 @@ pip install huggingface-hub
 huggingface-cli download <model> --local-dir <output_folder>
 ```
 
+### Using GGUF models
+
+To run any samples with a GGUF model, simply provide the path to the .gguf file in the `model_dir` parameter.
+
+This capability is currently available in preview mode and supports a limited set of topologies, including SmolLM, Qwen2.5. For other models 
```

Comment:
```suggestion
This capability is currently available in preview mode and supports a limited set of topologies, including SmolLM and Qwen2.5. For other models 
```
may be we a table of all supported models and this link we can provide. Please check

Reply:
I couldn't find a table of supported GGUF models. I took this information from [the OpenVINO documentation](https://docs.openvino.ai/2025/openvino-workflow-generative/inference-with-genai.html#inference-of-gguf-ggml-unified-format-models).
We should probably add information about the supported GGUF models to [the Supported Models page](https://openvinotoolkit.github.io/openvino.genai/docs/supported-models/), but I can't say for sure which models are fully supported.